src/bmp280/javaupm_bmp280.i

```diff
@@ -1,11 +1,20 @@
 %module javaupm_bmp280
+
+%import "../interfaces/javaupm_iTemperatureSensor.i"
+%import "../interfaces/javaupm_iHumiditySensor.i"
+%import "../interfaces/javaupm_iPressureSensor.i"
+
+
 %include "../upm.i"
+%include "cpointer.i"
+%include "typemaps.i"
+%include "arrays_java.i";
+%include "../java_buffer.i"
 
-%include "../interfaces/javaupm_iTemperatureSensor.i"
-%include "../interfaces/javaupm_iHumiditySensor.i"
-%include "../interfaces/javaupm_iPressureSensor.i"
+%typemap(javaimports) SWIGTYPE %{
+	import upm_interfaces.*;
+%}
 
-%include "bmp280_regs.h"
```

Comment:
Sorry about that. Changes on this file were done before you added that line in "src/bmp280/javaupm_bmp280.i", and I guess the file got committed without that line.
I will look over the changes to make sure that didn't happen anywhere else and create another PR